### DISCUSSION TITLE
Need help with spawning my entity.

### DISCUSSION THREAD

**Discussion by Quasar500** [bridge-core/bridge.#328] [Add-On Q&A] [answered]
So, I am creating a pack where every 30 seconds, an item appears in the players inventory. I have an entity(Giver) that has 0 health so it instant dies. Giver has a loot table with all the items you can get in your inventory. Now for the problem:

I need a way to summon Giver every 30 seconds on top of the player so it dies and drops the item. If possible I would also like it so there is no particles when it dies but that doesn't matter. (I am new to bridge so please answer with steps on what I have to do if you can)

![Screenshot (1)](https://user-images.githubusercontent.com/80136740/110190560-2661f680-7de1-11eb-89aa-5ee3fa9fead4.png)

**Comment by Chikorita-Lover** [comment, marked as answer]
You could use a behavior animation to summon the giver entity.
